OPSDOC.md

```diff
@@ -0,0 +1,64 @@
+# Nexus Repository Installer
+
+The purpose of this repository is to create Deb and Rpm packages of Nexus. 
+
+
+
+## Where?
+
+| What?           | Location                                                     |
+| --------------- | ------------------------------------------------------------ |
+| Source Code     | https://github.com/sonatype-nexus-community/nexus-repository-installer |
+| CI              | https://circleci.com/gh/sonatype-nexus-community/nexus-repository-installer |
+| Production Apt  | https://repo.sonatype.com/repository/community-apt-hosted/   |
+| Development Apt | https://nx-staging.sonatype.com/repository/community-apt-hosted/ |
+| Production Yum  | https://repo.sonatype.com/repository/community-yum-hosted/   |
+| Development Yum | https://nx-staging.sonatype.com/repository/community-yum-hosted/ |
+
+
+
+## How?
+
+The jobs are run via CircleCI. Below will be a sample overview of the build process. The trigger to run the job in CircleCI is done via scheduled trigger. This trigger runs and checks the latest version from https://download.sonatype.com. This is run daily on the master branch only.
```

Comment:
Interestingly(?) download.sonatype.com is forbidden to me.

Reply:
> Interestingly(?) download.sonatype.com is forbidden to me.

yeah it doesn't allow index browsing

Reply:
Cool.  Not sure if github auto links but if not, you could probably remove https:// unless there's some other reason.  The only reason I noticed was because I saw a link and clicked to see where it went tbh.

Reply:
Trivial suggestion implemented. Thanks!